media-gfx/gimp/gimp-2.99.14-r3.ebuild

```diff
@@ -134,7 +134,9 @@ src_prepare() {
 	sed -i -e 's/@PYTHON_PATH@/'${EPYTHON}'/' plug-ins/python/pygimp.interp.in || die
 
 	# Set proper intallation path of documentation logo
-	sed -i -e "s/'gimp-@0@'.format(gimp_app_version)/'gimp-@0@.@1@'.format(gimp_app_version, gimp_app_version_micro)/" data/images/meson.build || die
+	sed -i -e "s/'gimp-@0@'.format(gimp_app_version)/'gimp-${PVR}'/" data/images/meson.build || die
+
+	sed -i -e 's/#ifdef __gnu_linux__/#if defined (__gnu_linux__) \&\& defined (__GLIBC__)/' app/core/gimpbacktrace-backend.h || die #900148
```

Comment:
Please patch this instead, it's too easy for this to become stale.

Reply:
Should I also patch `data/images/meson.build` to provide version? It's depend on revision number.

Reply:
I now use prepared patch from https://github.com/gentoo/gentoo/pull/31087 - now this pull request depends on it.

Reply:
It's ok to sed meson.build for PVR because it changes based on some information from the ebuild.